.codegen/interface.go.tmpl

```diff
@@ -19,6 +19,9 @@ type {{.PascalName}}Service interface {
 	{{- end}}
     {{.PascalName}}(ctx context.Context{{if .Request}}, request {{.Request.PascalName}}{{end}}) {{ template "response-type" . }}
 	{{end -}}
+
```

Comment:
Nit: the `-}}` removes trailing whitespace, so it trims the newline in between.

Using just `{{ end }}` might work.